### PR TITLE
Amazonka and local stack support

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Comment by Jimbo4350** [comment]
You need to set the environment credentials for AWS:
```
           ┃   │ ━━━ Exception (AuthError) ━━━
           ┃   │ CredentialChainExhausted

```
I think they are:
```
AWS_ACCESS_KEY_ID="xxxx"
AWS_SECRET_ACCESS_KEY="xxxxx"
```

File: .github/workflows/haskell.yml [hunk 1]
```diff
@@ -67,10 +67,23 @@ jobs:
           archive-uri: https://cache.haskellworks.io/${{ env.CABAL_CACHE_VERSION }}/${{ runner.os }}/${{ matrix.cabal }}/${{ matrix.ghc }}
           skip: "${{ secrets.BINARY_CACHE_URI != '' }}"
 
+      - name: Unix packages
+        env:
+          LOCALSTACK_AUTH_TOKEN: ${{ secrets.LOCALSTACK_AUTH_TOKEN }}
```
Comment (Jimbo4350) [inline .github/workflows/haskell.yml]: Do we need the environment variables here too?

**Review by Jimbo4350 — APPROVED** [review]
LGTM! I think you can use terraform to deploy to the container to recreate our api

File: .github/workflows/haskell.yml (hunk 1 above)
Comment (newhoggy) [reply .github/workflows/haskell.yml]: Only the `LOCALSTACK_AUTH_TOKEN`

**Comment by newhoggy** [comment]
> AWS_ACCESS_KEY_ID="xxxx"
> AWS_SECRET_ACCESS_KEY="xxxxx"

Localstack doesn't need real credentials, only fake ones as long as they exist.  This is done programmatically now so the environment variables aren't needed anymore.

File: .github/workflows/haskell.yml [hunk 2]
```diff
@@ -18,10 +18,12 @@ jobs:
...
-        os: [ubuntu-latest, windows-latest]
-        include:
-          - ghc: "9.8.2"
-            os: macOS-latest
+        os:
+          - ubuntu-latest
+          # - windows-latest
+        # include:
+        #   - ghc: "9.8.2"
+        #     os: macOS-latest
```
Comment (newhoggy) [inline .github/workflows/haskell.yml:26]: Disabling these in the short term.

File: .github/workflows/haskell.yml [hunk 3]
```diff
@@ -71,6 +73,8 @@ jobs:
         run: cabal build all --enable-tests --enable-benchmarks
 
       - name: Test
+        env:
+          LOCALSTACK_AUTH_TOKEN: ${{ secrets.LOCALSTACK_AUTH_TOKEN }}
```
Comment (newhoggy) [inline .github/workflows/haskell.yml:77]: This is required to use the `localstack-pro`.

**Pull request merged**